### PR TITLE
Update dependencies

### PR DESCRIPTION
**Pull request by rsandell**
Now using Jenkins bom
Changed jackson dependencies to jackson api plugin dependency
The api plugin required a core bump

### CODE REVIEW

**Review by alecharp — APPROVED** [review]

**Review by MRamonLeon — APPROVED** [review]
@rsandell We also need to change the Jenkinsfile to run in the right Jenkins version

**Review by oleg-nenashev — APPROVED** [review]

**Comment by rsandell** [comment]
I fixed the Jenkinsfile, but I am not a committer so it won't take effect until merged.

**Pull request merged**
